<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(170, 0, 0, 0.9);
	border: none;
	border-radius: 8px;
	text-align: center;
	color: #FFFFFF;
}

QPushButton:hover {
	background-color: rgba(170, 0, 0,0.8);
}</string>
   </property>
   <property name="text">
    <string>Mesa 1</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(170, 0, 0, 0.9);
	border: none;
	border-radius: 8px;
	text-align: center;
	color: #FFFFFF;
}

QPushButton:hover {
	background-color: rgba(170, 0, 0,0.8);
}</string>
   </property>
   <property name="text">
    <string>Mesa 12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(170, 0, 0, 0.9);
	border: none;
	border-radius: 8px;
	text-align: center;
	color: #FFFFFF;
}

QPushButton:hover {
	background-color: rgba(170, 0, 0,0.8);
}</string>
   </property>
   <property name="text">
    <string>Mesa 8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>460</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(170, 0, 0, 0.9);
	border: none;
	border-radius: 8px;
	text-align: center;
	color: #FFFFFF;
}

QPushButton:hover {
	background-color: rgba(170, 0, 0,0.8);
}</string>
   </property>
   <property name="text">
    <string>Mesa 16</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(170, 0, 0, 0.9);
	border: none;
	border-radius: 8px;
	text-align: center;
	color: #FFFFFF;
}

QPushButton:hover {
	background-color: rgba(170, 0, 0,0.8);
}</string>
   </property>
   <property name="text">
    <string>Mesa 11</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(170, 0, 0, 0.9);
	border: none;
	border-radius: 8px;
	text-align: center;
	color: #FFFFFF;
}

QPushButton:hover {
	background-color: rgba(170, 0, 0,0.8);
}</string>
   </property>
   <property name="text">
    <string>Mesa 7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(170, 0, 0, 0.9);
	border: none;
	border-radius: 8px;
	text-align: center;
	color: #FFFFFF;
}

QPushButton:hover {
	background-color: rgba(170, 0, 0,0.8);
}</string>
   </property>
   <property name="text">
    <string>Mesa 15</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(170, 0, 0, 0.9);
	border: none;
	border-radius: 8px;
	text-align: center;
	color: #FFFFFF;
}

QPushButton:hover {
	background-color: rgba(170, 0, 0,0.8);
}</string>
   </property>
   <property name="text">
    <string>Mesa 10</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(170, 0, 0, 0.9);
	border: none;
	border-radius: 8px;
	text-align: center;
	color: #FFFFFF;
}

QPushButton:hover {
	background-color: rgba(170, 0, 0,0.8);
}</string>
   </property>
   <property name="text">
    <string>Mesa 9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(170, 0, 0, 0.9);
	border: none;
	border-radius: 8px;
	text-align: center;
	color: #FFFFFF;
}

QPushButton:hover {
	background-color: rgba(170, 0, 0,0.8);
}</string>
   </property>
   <property name="text">
    <string>Mesa 6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(170, 0, 0, 0.9);
	border: none;
	border-radius: 8px;
	text-align: center;
	color: #FFFFFF;
}

QPushButton:hover {
	background-color: rgba(170, 0, 0,0.8);
}</string>
   </property>
   <property name="text">
    <string>Mesa 14</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(170, 0, 0, 0.9);
	border: none;
	border-radius: 8px;
	text-align: center;
	color: #FFFFFF;
}

QPushButton:hover {
	background-color: rgba(170, 0, 0,0.8);
}</string>
   </property>
   <property name="text">
    <string>Mesa 5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(170, 0, 0, 0.9);
	border: none;
	border-radius: 8px;
	text-align: center;
	color: #FFFFFF;
}

QPushButton:hover {
	background-color: rgba(170, 0, 0,0.8);
}</string>
   </property>
   <property name="text">
    <string>Mesa 4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(170, 0, 0, 0.9);
	border: none;
	border-radius: 8px;
	text-align: center;
	color: #FFFFFF;
}

QPushButton:hover {
	background-color: rgba(170, 0, 0,0.8);
}</string>
   </property>
   <property name="text">
    <string>Mesa 3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(170, 0, 0, 0.9);
	border: none;
	border-radius: 8px;
	text-align: center;
	color: #FFFFFF;
}

QPushButton:hover {
	background-color: rgba(170, 0, 0,0.8);
}</string>
   </property>
   <property name="text">
    <string>Mesa 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_16">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>460</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(170, 0, 0, 0.9);
	border: none;
	border-radius: 8px;
	text-align: center;
	color: #FFFFFF;
}

QPushButton:hover {
	background-color: rgba(170, 0, 0,0.8);
}</string>
   </property>
   <property name="text">
    <string>Mesa 17</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	text-align: center;
	color: rgb(255,255,255);
	font-size: 16px;
	font: 75 16pt &quot;MS Shell Dlg 2&quot;;
}</string>
   </property>
   <property name="text">
    <string>Pizzaria D'França</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>0</y>
     <width>471</width>
     <height>631</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Downloads/pizza-g404793854_1920.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_5</zorder>
  <zorder>pushButton_6</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>pushButton_8</zorder>
  <zorder>pushButton_9</zorder>
  <zorder>pushButton_10</zorder>
  <zorder>pushButton_11</zorder>
  <zorder>pushButton_12</zorder>
  <zorder>pushButton_13</zorder>
  <zorder>pushButton_14</zorder>
  <zorder>pushButton_15</zorder>
  <zorder>pushButton_16</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
